pyquil/api/_qam.py

```diff
@@ -66,12 +67,17 @@ def write_memory(self, *, region_name: str, offset: int = 0, value=None):
 
         :param region_name: Name of the declared memory region on the QAM.
         :param offset: Integer offset into the memory region to write to.
-        :param value: Value to store at the indicated location.
+        :param value: Value(s) to store at the indicated location.
```

Comment:
Update documentation to describe the type of `value` and the dependant behaviour.